distribution/docs/src/main/resources/content/_reference/_appReferences/mg-spatial.adoc

```diff
@@ -47,6 +47,47 @@ index again.
 
 |===
 
+=== Standalone Solr Offline Gazetteer Feature
+
+The Standalone Solr Offline Gazetteer is meant to to be a reflection of the gazetteer data stored
+in the catalog while not being mixed in with all the other data. This allows the core to be
```

Comment:
:pencil: this might read clearer as `gazetteer data stored in the catalog but separated from other data`